service/raft/raft_group0.hh

```diff
@@ -18,6 +18,13 @@ namespace db { class system_keyspace; }
 
 namespace gms { class gossiper; class feature_service; }
 
+namespace raft {
```

Comment:
Why add it to the `raft` namespace?

Reply:
It's because of #21937 (that it can be moved into the "raft.hh" eventually and used also for the `config_member` attribute), so I'd like to already use that namespace to not have to change the occurrences later.

I guess I could also move it to the "raft.hh" right away, but it might look unnecessary in there if it will not be used there immediately (and only used in the "raf_group0.hh" in the voters interface at the moment).

Reply:
I am not sure if it worth it. config_member is an idl type, so it is not simply a matter of changing the type in the header.

Reply:
Yes, that's true, we'd also need to use it in the idl. It should generally work (I tried locally), but it causes a lot of changes across the codebase (also for serialization, de-serialization etc.). That's why I didn't want to do it right now but postpone for later. But it's true it is not entirely necessary (I created the ticket #21914 with P4).

What would be your preference in this case, to keep it in the service namespace?

Reply:
> 
> What would be your preference in this case, to keep it in the service namespace?

Yes. We can always move it later if we want to.

Reply:
Done, keeping it just in the `service` namespace.